### PR TITLE
Support security descriptor string

### LINKED ISSUE #329: Custom SDDL?
Is it possible to implement configuration for custom SDDL?
If so, where to start here?

I'm thinking about a simple config property that will just override default SDDL for created service.

### PR DESCRIPTION
**Pull request by NextTurn**
Fixes #329 

Hold for #392 

Possible property names:

- `<securtityDescriptor>`
- `<securtityDescriptorString>`
- `<securtityDescriptorSddl>`
- `<stringSecurtityDescriptor>`

### CODE REVIEW

File: src/Core/WinSWCore/ServiceDescriptor.cs [hunk 1]
```diff
@@ -698,5 +698,7 @@ public ProcessPriorityClass Priority
                 return (ProcessPriorityClass)Enum.Parse(typeof(ProcessPriorityClass), p, true);
             }
         }
+
+        public string? SecurityDescriptor => SingleElement("securityDescriptor", true);
```
Comment (oleg-nenashev) [inline src/Core/WinSWCore/ServiceDescriptor.cs]: reference the required format in the documentation?
https://docs.microsoft.com/en-us/windows/win32/secauthz/security-descriptor-definition-language

File: examples/sample-allOptions.xml [hunk 2]
```diff
@@ -80,6 +80,13 @@ Their modification will not take affect without the service re-installation.
   <resetfailure>1 hour</resetfailure>
   -->
 
+  <!--
+    OPTION: securityDescriptor
+    The security descriptor string in SDDL form.
```
Comment (oleg-nenashev) [inline examples/sample-allOptions.xml]: Link https://docs.microsoft.com/en-us/windows/win32/secauthz/security-descriptor-definition-language ?

**Comment by NextTurn** [comment]
But this PR isn't complete yet?

**Comment by oleg-nenashev** [comment]
Oh, sorry about that. Should I revert the merge?

**Comment by NextTurn** [comment]
I've added subsequent changes in #392.

**Pull request merged**